omod-common/src/main/java/org/openmrs/module/webservices/helper/TaskFactoryWrapper.java

```diff
@@ -0,0 +1,83 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+
+package org.openmrs.module.webservices.helper;
+
+import org.openmrs.scheduler.TaskDefinition;
+import org.openmrs.scheduler.SchedulerService;
+import org.openmrs.scheduler.SchedulerException;
+import org.openmrs.api.context.Context;
+
+import java.util.Collection;
+
+public class TaskFactoryWrapper {
```

Comment:
What is the main reason for setting up this class?

Reply:
I separated the Service actions from Resource and If I create the class like this,
Then only I can extend this class to MockTaskFactoryWrapper to mock the event for the tests.
(Actually, I followed the ModuleFactoryWrapper)

Reply:
Don't you think the name should change? Aren't factories for creating instances of objects?

Reply:
Class name has been changed to  TaskServiceWrapper